app/src/main/java/com/cornellappdev/android/eatery/ui/components/general/FilterRow.kt

```diff
@@ -334,25 +206,25 @@ fun FilterRowUpcoming(
 
         item {
             FilterButton(
-                onFilterClicked = { onFilterClicked(Filter.NORTH) },
-                selected = selectedFilters.contains(Filter.NORTH),
-                text = Filter.NORTH.text
+                onFilterClicked = { onFilterClicked(Filter.FromEatery.North) },
+                selected = selectedFilters.contains(Filter.FromEatery.North),
+                text = Filter.FromEatery.North.text
             )
         }
 
         item {
             FilterButton(
-                onFilterClicked = { onFilterClicked(Filter.WEST) },
-                selected = selectedFilters.contains(Filter.WEST),
-                text = Filter.WEST.text
+                onFilterClicked = { onFilterClicked(Filter.FromEatery.West) },
+                selected = selectedFilters.contains(Filter.FromEatery.West),
+                text = Filter.FromEatery.West.text
             )
         }
 
         item {
             FilterButton(
-                onFilterClicked = { onFilterClicked(Filter.CENTRAL) },
-                selected = selectedFilters.contains(Filter.CENTRAL),
-                text = Filter.CENTRAL.text
+                onFilterClicked = { onFilterClicked(Filter.FromEatery.Central) },
+                selected = selectedFilters.contains(Filter.FromEatery.Central),
+                text = Filter.FromEatery.Central.text
             )
```

Comment:
Can we make `FilterRowUpcoming` call `FilterRow`? Since it looks like you refactored `FilterRow` well but these other ones are still sus lol

I took a quick glance and it looks like they're similar enough to make this one depend on the general `FilterRow`?